### ISSUE TITLE
DataTable - dragging movements wcag 2.5.7

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7711]
Currently the resize columns functionality of the DataTable doesn't provide an alternative to dragging movements for mouse, touch screen, or stylus users. This violates WCAG rule 2.5.7.

Once this issue is resolved we should update the DataTable WCAG information on the design system site to set 2.5.7 to passing

**Comment by taysea** [comment]
Opening for WCAG docs
